src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -103,25 +282,21 @@
                     </Button>
                 </StackPanel>
 
-                <Popup x:Name="TraceValuePopup"
-                       Grid.Row="0"
-                       RenderTransformOrigin="0,1">
+                <Popup x:Name="TraceValuePopup">
                     <Popup.RenderTransform>
                         <TranslateTransform x:Name="TraceValuePopupTransform" X="50" Y="150"/>
                     </Popup.RenderTransform>
                     <Grid Background="{StaticResource ToolTipBackground}"
                           BorderBrush="{StaticResource ToolTipBorderBrush}"
-                          BorderThickness="3,3,3,3">
+                          BorderThickness="3">
                         <TextBlock x:Name="TraceValue"
```

Comment:
This pop up can now go off screen when hovering over the far right side of the graph, we will want to add in some logic to show it to the left in these cases. May want to do this in it's own PR though.

Reply:
I've replaced the Popup by a lighter control (only a Border instead of a Popup) + move the tooltip to the left or below the pointer when there is not enough space to display it.